css/src/components/segmented-control.scss

```diff
@@ -0,0 +1,135 @@
+$sc-padding-sm: 0.5em 1em !default;
+$sc-padding: 0.75em 1.25em !default;
+$sc-padding-lg: 1em 1.5em !default;
+
+$sc-font-color: $primary !default;
+$sc-border: $border-width solid $border !default;
+$sc-border-radius-rounded: $border-radius-rounded;
+$sc-hover-background-color: $primary-background !default;
+$sc-hover-border: $border-width solid $border !default;
+$sc-hover-border-color: $primary-hover !default;
+$sc-active-hover-background-color: $primary-hover !default;
+$sc-active-hover-border-color: $primary-hover !default;
+$sc-active-background-color: $primary !default;
+$sc-active-border-color: $primary !default;
+$sc-active-color: $text-invert !default;
+
+.segmented-controls {
+	display: grid;
+	justify-content: center;
+	width: 100%;
+	grid-gap: $spacer-3;
+	grid-template-columns: max-content auto max-content;
+	outline-color: $text; // Provided explicitly because outline-color isn't passing through tab-container's shadow DOM
+	font-size: $font-size-7;
+	white-space: nowrap;
+
+	.segmented-control-previous,
```

Comment:
Would suggest also an inner focus here on these elements. The outer focus on these buttons leads to some cut off outlines when these elements are the edge of the page:
<img width="79" alt="image" src="https://github.com/microsoft/atlas-design/assets/30843002/aad4c063-5f4e-4839-b9d9-8ce9f5355a88">

Reply:
@BenDMyers

Reply:
Ah, I had recommended keeping with our default focus there just to keep it as standard as possible - but I hadn't seen the clipped outlines.

I'm cool with inner focus for this reason.

Reply:
This should be adjusted now.